todo.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script type="text/javascript">
+</script>
+</head>
+
+<body>
+<h1>Todo List</h1>
+    <div id="todoList"></div>
+<hr>
+    <input type="text" id="todoDescription" placeholder="Add a todo here"/>
+    <button id="addTodo">Add</button>
+    
+    <div>Remaining tasks:<a id="remainingCount">0</a></div>
+    <div>Completed tasks:<a id="completedCount">0</a></div>
+<script >
+
+var totalTasks = 0;
```

Comment:
When you declare variables in a script tag they become global, which isn't what you want. One way you can get around this is by using a functional expression  `(function(){ .. })()`